docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1834 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #103.                    if len(out) &lt; k:
</pre><p>I don't really like this because it creates too much nested logic. Right now, we are doing <code>while</code> &gt; <code>if</code> &gt; <code>for</code> &gt; <code>if</code> , which I think is code smell. For each iteration of the <code>while</code> loop, I think it is fine to blindly append ALL discords (and allow over-appending) and then check in the <code>while</code> condition if the <code>len(out) &lt; k</code> . Then, outside of the <code>while</code> loop, perform the clean up.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>